singletons/utils.js

```diff
@@ -156,19 +156,75 @@ module.exports = (function (Module) {
 		 * Returns a formatted string, specifying an amount of time delta from current date to provided date.
 		 * @param {sb.Date|Date|number} target
 		 * @param {boolean} [skipAffixes] if true, the affixes "in X hours" or "X hours ago" will be omitted
+		 * @param {sb.Date} [deltaTo] If set, calculate time delta between target and deltaTo. If undefined, calculate
+		 * delta between target and the current time.
 		 * @returns {string}
 		 */
-		timeDelta (target, skipAffixes = false) {
-			const now = new sb.Date();
-			if (sb.Date.equals(now, target)) {
+		timeDelta (target, skipAffixes = false, deltaTo = undefined) {
```

Comment:
The issue states that the behaviour of ignoring leap days should be toggleable. If a new argument should be added, I'm afraid going up to 4 arguments is getting a little bit too dank. The alternative seems to be to turn the values into an `options` object-box, which in turn would require a rather large refactor, since the method is widely used. I think the latter solution should be the way taken.

Reply:
I missed the part that you wanted a flag. I'll work on adding the extra flag.